mmdet3d/datasets/pipelines/loading.py

```diff
@@ -238,6 +240,104 @@ def __repr__(self):
         return f'{self.__class__.__name__}(sweeps_num={self.sweeps_num})'
 
 
+@PIPELINES.register_module()
+class LoadCalibPoseTime(object):
+    """load calib poses and times for SemanticKitti dataset."""
```

Comment:
Please complement the docstring following the format of other pipelines.

Reply:
All the docstrings have been added.